### DISCUSSION TITLE
Question: How can I use a custom grid?

### DISCUSSION THREAD

**Discussion by egurapha** [Electrostatics/apbs#280] [Q&A]
Hello, I was wondering if it would be possible to specify my own set of grid points to compute the potentials at instead of using the center and fglen/cglen parameters. Basically, I would like to use a non-cubic grid if possible. Thanks!

**Comment by sobolevnrm** [comment]
Hello --

Do you want to evaluate the potential at locations other than the grid points determined by [fglen](https://apbs.readthedocs.io/en/latest/using/input/old/elec/fglen.html) or [cglen](https://apbs.readthedocs.io/en/latest/using/input/old/elec/cglen.html)?  If so, you can use the [multivalue](https://apbs.readthedocs.io/en/latest/using/tools.html) program.

If you want to adjust the center of the grid to shift the location of all grid points, you can use [fgcent](https://apbs.readthedocs.io/en/latest/using/input/old/elec/fgcent.html?highlight=fgcent).

Thank you
